scripts/run-tests.sh

```diff
@@ -5,4 +5,11 @@
 # To run a single test tests/test_foo.py::TestFoo::test_foo
 
 export PICCOLO_CONF="tests.piccolo_conf"
-python -m pytest --cov=piccolo_admin --cov-report xml --cov-report html --cov-fail-under 23 -s $@
+cd admin_ui
+npm install
+npm run build
+rm -rf node_modules
```

Comment:
You're right to do this, as Jinja freaks out if it can't find a template.

Building the front end is quite a time consuming process though. An alternative might be to create the dist folder, and put a dummy index file in there.

```
mkdir piccolo_admin/dist
touch piccolo_admin/dist/index.html
```

Reply:
Great idea. I don't have time now but I'll try later

Reply:
This works
```
mkdir piccolo_admin/dist
mkdir piccolo_admin/dist/css
mkdir piccolo_admin/dist/js
touch piccolo_admin/dist/index.html
python -m pytest --cov=piccolo_admin --cov-report xml --cov-report html --cov-fail-under 85 -s $@
rm -rf piccolo_admin/dist
```

Reply:
Cool, will try and finish reviewing this today.